megamek/src/megamek/common/Aero.java

```diff
@@ -154,8 +154,12 @@ public String[] getLocationNames() {
     // fixed and pod-mounted.
     private int podHeatSinks;
 
-    protected int maxBombPoints = 0;
-    protected int[] bombChoices = new int[BombType.B_NUM];
+    protected int maxIntBombPoints = 0;
```

Comment:
(not about this line) 

Since fighters are no longer directly Aero, do we ever instantiate Aero? It seems that the Aero class could be abstract.

Reply:
> (not about this line)
> 
> Since fighters are no longer directly Aero, do we ever instantiate Aero? It seems that the Aero class could be abstract.

It could be, although I wasn't planning to at this juncture.  It would let us move a lot of stuff out of Aero.